debian/control

```diff
@@ -1,9 +1,9 @@
 Package: dokku
-Version: 0.3.12
+Version: 0.4.3
```

Comment:
Note: this actually never *needs* to be bumped, as the deb.mk script uses sed to fix it in the package. But no need to change here :)